.github/workflows/publish.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         #    https://github.com/jupyterhub/configurable-http-proxy/settings/secrets/actions
         if: startsWith(github.ref, 'refs/tags/')
         run: |
-          docker login -u "${{ secrets.DOCKERHUB_USERNAME }}" -p "${{ secrets.DOCKERHUB_TOKEN }}"
+          docker login -u "${{ secrets.QUAY_USERNAME }}" -p "${{ secrets.QUAY_PASSWORD }}" quay.io
```

Comment:
This changes the creds, but doesn't change where it's pushed (next step in gettags needs `quay.io` added, I think)

Reply:
@minrk how about now?

Reply:
I think that's right, thanks.